test/src/utils/check.rs

```diff
@@ -0,0 +1,30 @@
+use crate::Node;
+use ckb_jsonrpc_types::Status;
+use ckb_types::core::TransactionView;
+
+pub fn is_transaction_pending(node: &Node, transaction: &TransactionView) -> bool {
+    node.rpc_client()
+        .get_transaction(transaction.hash())
+        .map(|txstatus| txstatus.tx_status.status == Status::Pending)
+        .unwrap_or(false)
+}
+
+pub fn is_transaction_proposed(node: &Node, transaction: &TransactionView) -> bool {
+    node.rpc_client()
+        .get_transaction(transaction.hash())
+        .map(|txstatus| txstatus.tx_status.status == Status::Proposed)
+        .unwrap_or(false)
+}
+
+pub fn is_transaction_committed(node: &Node, transaction: &TransactionView) -> bool {
+    node.rpc_client()
+        .get_transaction(transaction.hash())
+        .map(|txstatus| txstatus.tx_status.status == Status::Committed)
+        .unwrap_or(false)
+}
+
+pub fn is_transaction_unknown(node: &Node, transaction: &TransactionView) -> bool {
+    node.rpc_client()
+        .get_transaction(transaction.hash())
+        .is_none()
+}
```

Comment:
how about move this file into `test/src/assertion/` dir

Reply:
We do agree that `test/src/assertion` and `check.rs` have many duplicate codes.
I would like to remove the utils in `test/src/assertion` or move into `test/src/utils/assertion.rs`. But to avoid more changes in this PR, I didn't do that.

// Maybe utils in `test/src/assertion` are good for code reuse, but less readability.